libraries/YarpPlugins/TechnosoftIpos/IPositionControl2RawImpl.cpp

```diff
@@ -85,7 +85,8 @@ bool roboticslab::TechnosoftIpos::relativeMoveRaw(int j, double delta)
     uint8_t msg_position_target[]= {0x23,0x7A,0x60,0x00,0x00,0x00,0x00,0x00}; // Position target
 
     //-- 11.38 = ( 4 * 1024 pulse / 360 deg ) // deg -> pulse = UI (pos)
-    int sendDelta = delta * this->tr * 11.38;  // Appply tr & convert units to encoder increments
+    double val = encoderPulses / 360; // encoderPulses: value encompasses the pulses-per-slot factor (usually 4) and number of total slots of the encoder (currently: 4 * 1024).
```

Comment:
`encoderPulses / 360.0` (note the decimal point) so that the result is a double. See https://stackoverflow.com/q/7571326. Same a few lines below.